bedrock/firefox/templates/firefox/australis/base.html

```diff
@@ -54,5 +60,10 @@
 {% block email_form %}{% endblock %}
 {% block js %}{% endblock %}
 
-{# Bug 1381776 #}
-{% block update_notification %}{% endblock %}
+{# Bug 1387984 #}
+{% block update_notification %}
+  {% if switch('firefox-update-notification-modal', ['en-US']) %}
```

Comment:
This PR uses a different switch name to the regular notification, so it can be toggled on/off independently. It will need enabling once this merges/goes to prod.